packages/react-native/React/Base/RCTUIKit.h

```diff
@@ -417,6 +417,11 @@ CGPathRef UIBezierPathCreateCGPathRef(UIBezierPath *path);
  */
 @property (nonatomic, assign) BOOL enableFocusRing;
 
+#if TARGET_OS_OSX // [macOS
+// macOS Properties
+@property (nonatomic, assign) CATransform3D transform3D;
+#endif // macOS]
```

Comment:
Isn't this already in a giant macOS block, so we don't need the extra ifdef?

Reply:
True. Will fix.